ReactAndroid/src/main/java/com/facebook/react/views/scroll/ReactHorizontalScrollView.java

```diff
@@ -27,11 +27,16 @@
   private boolean mDragging;
   private boolean mFlinging;
   private boolean mDoneFlinging;
+  private boolean mScrollEnabled;
```

Comment:
I think I would feel more comfortable if this were explicitly set to true.  Does ReactNative always call us if there is a default property?  Or is it always set in the js?  It doesn't look like it.